scrabble.js

```diff
@@ -1,8 +1,144 @@
-var Scrabble = function() {};
+Scrabble =  {
+  scoreChart: {
+    1: ["A","E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2: ["D", "G"],
+    3: ["B", "C", "M", "P"],
+    4: ["F", "H", "V", "W", "Y"],
+    5: ["K"],
+    8: ["J", "X"],
+    10: ["Q", "Z"]
+  },
 
-// YOUR CODE HERE
-Scrabble.prototype.helloWorld = function() {
-  return 'hello world!';
+
+  score: function(word) {
+    var score = 0;
+    var upWord = word.toUpperCase();
+    var num = Object.keys(Scrabble.scoreChart);
+    var letters = Object.values(Scrabble.scoreChart);
+    for (i = 0; i < upWord.length; i++) {
```

Comment:
Because you don't have the `var` in front of these iteration variables, each of these loops are creating global variables